### ISSUE TITLE
[SF][A11y][ActionSheet]: border color fails to meet 3:1 against its inner white background in sap_horizon theme

### ISSUE THREAD

**Issue by GongRichard** [SAP/ui5-webcomponents-react#3229]
**Describe the bug**
border color fails to meet 3:1 against its inner white background in sap_horizon theme

**To Reproduce**
1. Go to 'https://sap.github.io/ui5-webcomponents-react/?path=/story/modals-popovers-actionsheet--default-story&globals=theme:sap_horizon'
2. Click on 'Open ActionSheet'
3. Use CCA tool to compare the color

**Expected behavior**
Popovers - The border has to meet 3:1 CC
https://wiki.wdf.sap.corp/wiki/display/Acc/Color+Contrasts+for+Non-Textual+Information#ColorContrastsforNonTextualInformation-Popover,Popup

**Screenshots**
<img width="1386" alt="image" src="https://user-images.githubusercontent.com/13098801/178265283-0a5ee59e-6dfb-432c-a2c4-6855865d6cc6.png">

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: latest
`@ui5/webcomponents-react` version: latest
Operating System:
Browser:

**Additional context**

**Comment by Lukas742** [comment]
Hi @GongRichard 

the `ActionSheet` doesn't have a special background, box-shadow or border color in desktop mode. Could you verify if the same issue is also applicable for the `Popover` component and if not tell us which elements (preferably with code examples) don't meet the contrast criteria?

**Comment by GongRichard** [comment]
Hi @Lukas742 ,
The box-shadow and background were defined by ui5-responsive-popover. Maybe you can forward this issue to ui5 wc team.

<img width="1791" alt="image" src="https://user-images.githubusercontent.com/13098801/178493307-7dee29e7-a8a2-4b62-825a-3a10ea6c5111.png">

Thanks,
Richard

**Comment by Lukas742** [comment]
Will do, thanks :) 

Hi colleagues, 
could you please take a look at this issue. It seems like the box-shadow of the ResponsivePopover/Popover doesn't meet the contrast criteria in `sap_horizon` theme.
